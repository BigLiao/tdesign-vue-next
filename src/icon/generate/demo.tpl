<template>
  <!--
    该文件有脚本 tdesign/tdesign-web-vue/src/icon/generate/index.js 自动生成。
    若需要编辑，请编辑源文件：tdesign/tdesign-web-vue/src/icon/generate/demo.tpl 。
  -->
  <div class="t-demo-iconfont">
    <p>目前设计师给到的全部图标</p><br>
    <%= icons %>
  </div>
</template>

<style lang="less">
  .t-demo-iconfont {
    .t-demo-icon {
      width: 130px;
      height: 100px;
      margin-right: 24px;
      display: inline-block;
      text-align: center;
      vertical-align: top;
      .t-icon {
        font-size: 25px;
        margin-right: 0;
        color: #999999;
      }
      .t-icon:hover {
        color: #0052d9;
        font-size: 30px;
      }
      .t-demo-icon-name {
        text-align: center;
        margin-top: 8px;
        font-size: 12px;
      }
    }
  }
</style>
