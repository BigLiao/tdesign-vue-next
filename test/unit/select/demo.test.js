import { mount } from '@vue/test-utils';
import demo from '@/examples/select/demos/base.vue';

// unit test for component in examples.
describe('Select', () => {
  it('base demo works fine', () => {
    const wrapper = mount(demo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
