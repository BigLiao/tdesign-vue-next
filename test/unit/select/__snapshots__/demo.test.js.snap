// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select base demo works fine 1`] = `
<div>
  <div
    class="demo-select-base"
  >
    <div
      class="t-popup-reference"
      style="width: 100%; position: relative;"
    >
       
      <div
        class="t-select"
      >
        <svg
          class="t-icon t-icon-undefined t-select-left-icon"
          style="display: none;"
        >
          <use
            xlink:href="#t-icon-undefined"
          />
        </svg>
         
        <span
          class="t-select-placeholder"
        >
           请选择
        </span>
         
        <input
          name="t-select"
          style="width: 0px;"
          type="hidden"
        />
          
        <!---->
         
        <div
          class="t-input t-select-input"
          style="display: none;"
        >
          <input
            class="t-input__inner"
            placeholder="请选择"
            type="text"
          />
        </div>
         
        <svg
          class="t-icon t-icon-chevron-down t-select-right-icon t-is-visible"
        >
          <use
            xlink:href="#t-icon-chevron-down"
          />
        </svg>
         
        <svg
          class="t-icon t-icon-close t-select-right-icon"
          style="display: none;"
        >
          <use
            xlink:href="#t-icon-close"
          />
        </svg>
         
        <svg
          class="t-icon t-icon-loading t-select-right-icon"
          style="display: none;"
        >
          <use
            xlink:href="#t-icon-loading"
          />
        </svg>
         
        <!---->
      </div>
       
      <div
        class="resize-sensor"
        dir="ltr"
        style="pointer-events: none; position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px; overflow: hidden; z-index: -1; visibility: hidden; max-width: 100%;"
      >
        <div
          class="resize-sensor-expand"
          style="pointer-events: none; position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px; overflow: hidden; z-index: -1; visibility: hidden; max-width: 100%;"
        >
          <div
            style="position: absolute; left: 0px; top: 0px; transition: 0s;"
          />
        </div>
        <div
          class="resize-sensor-shrink"
          style="pointer-events: none; position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px; overflow: hidden; z-index: -1; visibility: hidden; max-width: 100%;"
        >
          <div
            style="position: absolute; left: 0px; top: 0px; transition: 0s; width: 200%; height: 200%;"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
